google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -115,8 +116,23 @@ def _send_email(self, summary_data, details_data):
         email_summary_config = (
             self.notifier_config.get('inventory').get('email_summary'))
 
-        email_util = email.EmailUtil(
-            email_summary_config.get('sendgrid_api_key'))
+        if self.notifier_config.get('email_connector_config'):
+            email_connector_config_auth = (
+                self.notifier_config.get('email_connector_config').get('auth'))
+
+            email_util = SendgridConnector(self.notifier_config
+                                           .get('email_connector_config')
+                                           .get('sender'),
+                                           self.notifier_config
+                                           .get('email_connector_config')
+                                           .get('recipient'),
+                                           email_connector_config_auth)
+        # else block below is added for backward compatibility.
+        else:
+            email_util = SendgridConnector(email_summary_config.get('sender'),
+                                           email_summary_config.get('recipient')
+                                           ,
```

Comment:
This should not be a dangling comma.  If the line is too long, you can wrap like this:
```
email_util = SendgridConnector(
    email_summary_config.get('sender'),
    email_summary_config.get('recipient'),
    email_summary_config)
```

Reply:
Done